dandiapi/api/mail.py

```diff
@@ -66,3 +76,44 @@ def send_ownership_change_emails(dandiset, removed_owners, added_owners):
     messages += [build_added_message(dandiset, added_owner) for added_owner in added_owners]
     with mail.get_connection() as connection:
         connection.send_messages(messages)
+
+
+# Email sent to the DANDI list when a new user logs in for the first time
+
+
+def registered_subject(sociallogin):
+    return f'DANDI: New user registered: {sociallogin.user.email}'
+
+
+def registered_message(sociallogin):
+    return (
+        f'A new user has logged in to the DANDI application: {sociallogin.user.email}. '
+        f'Their GitHub login is {sociallogin.account.extra_data["login"]}.'
+    )
```

Comment:
```
Dear {sociallogin.account.extra_data["name"]} (Github ID: {sociallogin.account.extra_data["login"]}),

Welcome to DANDI. 

You are now registered on the DANDI archive. Registering allows you to create Dandisets and upload data right away. You can also use the Jupyterhub (https://hub.dandiarchive.org) for computing on dandisets in the cloud. 

It may take up to 24 hours for your hub account to be activated and for your email to be registered with our Slack workspace.

Please use the following links to post any questions or issues. 

Discussions: https://github.com/dandi/helpdesk/discussions 
Issues: https://github.com/dandi/helpdesk/issues

Thank you for choosing DANDI for your neurophysiology data needs.

Sincerely,

The DANDI team
```

Reply:
These registration emails are currently only going to `dandi@mit.edu` so that we are aware of new registrants. I assume you would also like this email sent to the user.

Reply:
yes.